scout/parse/config_base.py

```diff
@@ -0,0 +1,262 @@
+"""Class to hold information about scout load config"""
```

Comment:
Perhaps renaming this file to something more self-explanatory, for instance "case_config_base" ?